client/python/samples/utils/vehicle_utils.py

```diff
@@ -25,15 +25,15 @@ def vehicle_postprocess(response, image_path, output_name, output_save_path):
     width = image.shape[1]
     height = image.shape[0]
     for i in range(output.shape[2]):
-        image = draw_vehicle(output[0, 0, i, ...], image, width, height)
+        image = draw_bounding_box(output[0, 0, i, ...], image, width, height, 0.5)
```

Comment:
i would still assign this `0.5` to a variable to make it more visible what this value is about